relation_engine_server/utils/arango_client.py

```diff
@@ -9,6 +9,20 @@
 from relation_engine_server.utils.config import get_config
 
 _CONF = get_config()
+adb_url = _CONF["api_url"]
+auth = (_CONF["db_user"], _CONF["db_pass"])
+
+
+def adb_request(req_method, url_append, **kw):
+    """Make HTTP request to ArangoDB server"""
+    resp = req_method(
+        adb_url + url_append,
+        auth=auth,
+        **kw,
+    )
```

Comment:
are there any other `arango_client` functions that can use this shorthand or are the others more complex?

Reply:
I think this code pattern is used like a bajillion times ... very repetitive ... but I only DRYed it up in the functions I touched